GPU/Common/TextureCacheCommon.cpp

```diff
@@ -947,23 +952,21 @@ bool TextureCacheCommon::MatchFramebuffer(
 			(fb_format == GE_FORMAT_DEPTH16 && entry.format == GE_TFMT_CLUT16) ||
 			(fb_format == GE_FORMAT_DEPTH16 && entry.format == GE_TFMT_5650) ||
 			(fb_format == GE_FORMAT_8888 && entry.format == GE_TFMT_CLUT32) ||
-			(fb_format != GE_FORMAT_8888 && entry.format == GE_TFMT_CLUT16);
+			(fb_format != GE_FORMAT_8888 && entry.format == GE_TFMT_CLUT16) ||
+			(fb_format == GE_FORMAT_8888 && entry.format == GE_TFMT_CLUT8);  // Mixed byte size, format, reinterpret in depal!
```

Comment:
This isn't actually supported yet, though, right?

-[Unknown]

Reply:
Oops, that was supposed to be part of a commit in the next PR.